src-tauri/Cargo.toml

```diff
@@ -17,7 +17,7 @@ tauri-build = { version = "1.1.1", features = [] }
 [dependencies]
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
-tauri = { version = "1.1.1", features = ["api-all", "updater"] }
+tauri = { version = "1.1.1", features = ["api-all"] }
```

Comment:
Might wanna double check the files you commit. Had the same issue when I disabled the updater locally at some point ^^

Same with the removed `src-tauri/Cargo.lock`